api/src/main/java/org/openmrs/api/impl/PersonServiceImpl.java

```diff
@@ -488,18 +488,9 @@ public Relationship saveRelationship(Relationship relationship) throws APIExcept
 	 */
 	@Override
 	public Relationship voidRelationship(Relationship relationship, String voidReason) throws APIException {
-		if (relationship.getVoided()) {
-			return relationship;
-		}
```

Comment:
Did you change this accidentally?

Reply:
@dkayiwa, sorry about that earlier mess. Intelij sometimes pays deadly games. But its solved now slightly_smiling_face